### PR TITLE
Fix cli package resolution for `@astrojs/db`

### PR DESCRIPTION
**Pull request by BryceRussell**
## Changes

Fixes bug where every time a `astro db` command is used it prompts the user to add `@astrojs/db` even if it is already installed

## Testing

No tests, it is a pretty small fix for an error that doesn't get surfaced because it is inside a try/catch

> Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only URLs with a scheme in: file, data, and node are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'

## Docs

No docs needed

### CODE REVIEW

**Review by bluwy — APPROVED** [review]

**Review by ematipico — APPROVED** [review]

**Comment by BryceRussell** [comment]
> "We are addressing this!"

This was mentioned in Discord, maybe the fix for this is more complicated? I will close this if I hear anything else about the issue being fixed

**Review by matthewp — APPROVED** [review]

**Comment by bluwy** [comment]
That is being fixed at https://github.com/withastro/astro/pull/10342. It looks like this is a bit different than that though.

I was waiting for someone more familiar with db before merging this, just in case, but seems like this is a simple fix enough.

**Comment by BryceRussell** [comment]
Now that this has been approved, is this waiting on me to merge it? Sorry I haven't made a PR in a long time and I just want to make sure I am not blocking this

**Comment by bluwy** [comment]
You can merge but no I didn't expect that and it wasn't blocking the PR. I saw Matthew approved and I thought he was gonna merge it. I'm gonna merge this now 😄

**Comment by BryceRussell** [comment]
Awesome thanks!

**Pull request merged**
